src/Controller/Admin/ParticipationController.php

```diff
@@ -12,53 +12,28 @@
 namespace App\Controller\Admin;
 
 use AlterPHP\EasyAdminExtensionBundle\Controller\EasyAdminController;
-use App\Entity\Conference;
 use App\Entity\Participation;
 use App\Enum\Workflow\Transition\Participation as ParticipationTransition;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\Workflow\Registry as WorkflowRegistry;
 
-class AdminController extends EasyAdminController
+class ParticipationController extends EasyAdminController
 {
     public function __construct(
         private WorkflowRegistry $workflowRegistry,
     ) {
     }
 
-    public function excludeConferenceAction(): Response
+    protected function createNewParticipationEntity(): Participation
     {
-        /** @var Conference $conference */
-        $conference = $this->request->attributes->get('easyadmin')['item'];
-        $conference->setExcluded(true);
-
-        $this->em->flush();
-
-        return $this->redirectToRoute('easyadmin', [
-            'action' => 'list',
-            'entity' => $this->request->query->get('entity'),
-            'page' => $this->request->query->get('page'),
-        ]);
-    }
-
-    public function includeConferenceAction(): Response
-    {
-        /** @var Conference $conference */
-        $conference = $this->request->attributes->get('easyadmin')['item'];
-        $conference->setExcluded(false);
-
-        $this->em->flush();
-
-        return $this->redirectToRoute('easyadmin', [
-            'action' => 'list',
-            'entity' => $this->request->query->get('entity'),
-            'page' => $this->request->query->get('page'),
-        ]);
+        return new Participation($this->getUser());
     }
 
     #[Route(path: '/participation/accept', name: 'participation_accept')]
-    #[Security('ROLE_ADMIN')]
+
+    /** @IsGranted("ROLE_ADMIN") */
```

Comment:
The attribute is not working with security it seems.

The blank line is added by the CS-fixer, so I'm not sure about what to do with it

Reply:
If it's the case, it's a bug in symfony and it must be reported
about the blank line: it's a bug in php cs fixer, and it must be reported too

Reply:
After double checking :
The attribute is working fine (it was just not working with `#[Security]`)

The CS-fixer adds a blank line only if there is an annotation between the attribute and the method.
```
#[IsGranted('ROLE_ADMIN')]
/** @return Response */
```

This will add a blank line between them. This is however not the case if the return annotation is first.
Should I report it ?

Reply:
> The attribute is working fine (it was just not working with `#[Security]`)

IMHO, this is because you misused the attritube. it must be something like `#[Security("is_granted('FOOBAR')")]`. Anyway



> This will add a blank line between them. This is however not the case if the return annotation is first.

yes you can, it will help other devs.
But in our case, just drop the `/** @return Response */`, and add a real type hint. PHP 7 FTW (I know, we are using 8.0)

Reply:
It's already the case, it was just for testing purposes :)